Kronos-Shifts-Connector/Microsoft.Teams.Shifts.Integration/Microsoft.Teams.App.KronosWfc.BusinessLogic/Shifts/IShiftsActivity.cs

```diff
@@ -69,7 +70,10 @@ public interface IShiftsActivity
         /// <param name="kronosId">The id of the employee.</param>
         /// <param name="startTime">The start time of the shift.</param>
```

Comment:
do these want to be reworded slightly to make it clear that this will be the new start time after it is edited?